common/shlibs

```diff
@@ -352,7 +352,7 @@ libpoppler-cpp.so.0 poppler-cpp-0.18.2_1
 libpoppler-qt5.so.1 poppler-qt5-0.31.0_1
 libtcl8.6.so tcl-8.6.0_1
 libtk8.6.so tk-8.6.1_3
-libsqlite3.so.0 sqlite-3.38.0_1
+libsqlite3.so.0 sqlite-3.38.1_1
```

Comment:
Why is this necessary?

Reply:
I was trying to avoid making it lag behind. 
If it's unnecessary I can revert this bit.

Reply:
common/shlibs list shlibs and packages in which they are in versions in which they first appeared. `xbps-src` will scan the package for required shlibs, look at this file and create a list of dependencies. Without this change, a new package/an updated one will depend on sqlite>=3.38.0, with this change it will depend on sqlite>=3.38.1.
If there is no good reason to do this, we shouldn't do it. This makes reverts hard and prevents partial updates. It would make sense if sqlite added some new symbols and packages built with this version would crash with the old one. Then we would want new packages (built with new sqlite) to also depend on the new sqlite version, but that's not this case.